test/models/media_test.rb

```diff
@@ -17,7 +17,7 @@ class MediaTest < ActiveSupport::TestCase
     assert_equal 'Porta dos Fundos', data['title']
     assert_equal 'portadosfundos', data['username']
     assert_equal 'Porta dos Fundos', data['author_name']
-    assert_equal 'user', data['subtype']
+    assert_equal 'channel', data['subtype']
```

Comment:
I didn't understand why the `subtype` is returning `channel`. The URL is https://www.youtube.com/user/portadosfundos and I verified that YouTube doesn't redirect to https://www.youtube.com/channel/portadosfundos. And the `subtype` is extracted from the URL.

Reply:
Yes, need to dig deeper into this one.

Reply:
Noting that the Pender API returns a url of `https://www.youtube.com/channel/UCEWHPFNilsT0IfQfutVzsag` - I expect this is the canonical URL that is sniffed by Pender on the profile page, and the `subtype` is extracted afterwards. I am not even sure what `subtype` is supposd to represent :-)

Reply:
The idea was to differentiate between "people" and "pages"... but I don't think it has any practical implications :)

Reply:
OK in that case I am merging :-)